<launch>
    <node pkg="can_receive" type="can_receive_node" name="can_receive_node" output="screen"/>
    <node pkg="prediction_kf" type="prediction_kf_node_global" name="prediction_kf_node_global" output="screen">
        <param name="attitude_topic" type="string" value="/can_receive_1/attitude"/>
        <param name="real_visual_topic" type="string" value="/detected_armor"/>
        <param name="publisher_topic" type="string" value="/prediction_kf/predict"/>
        <param name="debug_topic" type="string" value="/prediction_kf/preprocessed"/>
        <param name="transform_topic" type="string" value="/prediction_kf/transformed"/>
        <param name="R_pos" type="double" value="16.0"/>
        <param name="Q_pos" type="double" value="1.0"/>
        <param name="Q_vel" type="double" value="16.0"/>
        <param name="P_matrix_weight" type="double" value="65.0"/>
        <param name="chi_square_threshold" type="double" value="100000.0"/>
        <param name="outlier_l2_norm_ratio" type="double" value="1.5"/>
        <param name="pitch_delay" type="double" value="0"/>
        <param name="yaw_delay" type="double" value="0"/>
        <param name="repropagate_time" type="int" value="0"/>
    </node>
</launch>
